src/less/video/base/video.less

```diff
@@ -56,3 +56,7 @@
     background-color: rgba(0, 0, 0, 0.7);
     border-radius: 8px;
 }
+
+.shaka-scrim-hide .shaka-scrim-container {
```

Comment:
Is `.shaka-scrim-hide` something provided by the shaka CSS? Or something that you added?

Reply:
Something I added because I needed a new concise class I could toggle. 

It could probably use a better name though 😂

Reply:
How about a more state based selector, like `.video-player--poster .shaka-scrim-container`.

We could also do: `.video-player video[poster] ~ .shaka-controls-container .shaka-scrim-container` but might be a little too crazy/brittle.

Reply:
it's weird trying to target the video's poster, it seems like you have to do it from the top. It's strange to me that they didn't provide, in the `video` tag, some sort of way to target the poster's styles more easily. 

The issue with `.video-player--poster` is that it's not really targeting the poster through the CSS, it's the JS that's removing the styles when the poster happens to be displayed.

Reply:
So I feel like that name would be sort of misleading

Reply:
Not sure I follow. The easiest thing to do would be to just toggle the `hidden` property directly on the element that you want to hide with JavaScript.

A class like `.shaka-scrim-hide` isn't that helpful/useful. If you are wanting to do it via a CSS class then a class like `video--poster` or `video--poster-mode` gives a clue as to what state the video is in. So:

```css
.video-player--poster .shaka-scrim-container {
   display: none;
}
```

This lets us add other styles while in poster mode.

Another option is adding a BEM child class of `video-player__control` to the `video` element and then going with:

```
.video-player__control[poster] ~ .shaka-controls-container .shaka-scrim-container {
  display: none;
}
```

Reply:
So I agree that toggling `hidden` would probably be the ideal. But in this case I'm not sure in what lifecycle method in Marko I would set up a querySelector to find the `shaka-scrim-container` to hide it.

Reply:
also for some reason `display: none` doesn't let me bring the scrim container back for the video, so it does need to be `visibility: hidden`

Reply:
I changed the class' name to `video-player--poster`